src/Stryker.Core/Stryker.Core/Options/StrykerInputs.cs

```diff
@@ -86,6 +87,7 @@ public StrykerInputs(IFileSystem fileSystem = null)
         public MutateInput MutateInput { get; init; } = new();
         public IgnoreMethodsInput IgnoredMethodsInput { get; init; } = new();
         public IgnoreMutationsInput ExcludedMutationsInput { get; init; } = new();
+        public IgnoreLinqExpressionInput ExcludedLinqExpressionInput { get; init; } = new();  
```

Comment:
The Input is never filled. Please see our CLI project. You need to add a line to https://github.com/stryker-mutator/stryker-net/blob/v1.0/src/Stryker.CLI/Stryker.CLI/JsonConfigHandler.cs 

That will fill the input and actually use user input. Please also add a test for that. I'd like to actually see this feature used in https://github.com/stryker-mutator/stryker-net/blob/v1.0/src/Stryker.CLI/Stryker.CLI.UnitTest/filled-stryker-config.json and test if the value is set in the input. That way we can be sure the value is read from the json

Reply:
I changed the ConfigFileTests.WithConfigFile_ShouldStartStrykerWithConfigFileOptions  Tests
adding :
```csharp
actualInputs.ExcludedMutationsInput.SuppliedInput.ShouldContain("linq.FirstOrDefault");
````